dataset.py

```diff
@@ -61,7 +61,7 @@ def dist_to_left_dot(self, x, y):
         return np.sqrt((x - 0.5 * self.r_big)**2 + (y - self.r_big)**2)
 
     def __getitem__(self, index):
-        sample = (self.__vals[index], self.__cs[index])
+        sample = (self.__vals[index].copy(), self.__cs[index])
```

Comment:
maybe also copy the label? who knows, maybe there is a transform that somehow changes the label...e.g. into a one-hot vector or something like that?

Reply:
yes, in principle you are right, but since here labels are `int`s, which are immutable, there's no danger of modifying them.

Reply:
ah...right!